### PR TITLE
[SW-684] Update spot_ros2 metadata

### PR DESCRIPTION
**Pull request by tcappellari-bdai**
## Change Overview

I added package descriptions to the packages missing one and standardized all the maintainer and license info

## Testing Done

N/A

### CODE REVIEW

**Comment by tcappellari-bdai** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #270** <a href="https://app.graphite.dev/github/pr/bdaiinstitute/spot_ros2/270?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/bdaiinstitute/spot_ros2/270?utm_source=stack-comment).

File: spot_description/package.xml [hunk 1]
```diff
@@ -3,9 +3,9 @@
 <package format="3">
   <name>spot_description</name>
   <version>0.0.0</version>
-  <description>TODO: Package description</description>
-  <maintainer email="maximillian.kirsch@alumni.fh-aachen.de">max</maintainer>
-  <license>TODO: License declaration</license>
+  <description>This package contains the URDF files for Spot</description>
+  <maintainer email="engineering@theaiinstitute.com">BD AI Institute</maintainer>
```
Comment (amessing-bdai) [inline spot_description/package.xml:7]:
```suggestion
  <maintainer email="engineering@theaiinstitute.com">The AI Institute</maintainer>
```

File: spot_msgs/LICENSE [hunk 2]
```diff
@@ -1,3 +1,26 @@
+MIT License
+
+Copyright (c) 2023 Boston Dynamics AI Institute
```
Comment (amessing-bdai) [inline spot_msgs/LICENSE:3]:
```suggestion
Copyright (c) 2023-2024 Boston Dynamics AI Institute
```

**Review by khughes-bdai — CHANGES_REQUESTED** [review]
Noticed there are still some missing licenses in the packages in the `examples` directory, could those be added too?

**Review by khughes-bdai — APPROVED** [review]
LGTM

File: examples/inverse_kinematics_example/package.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@
   <name>inverse_kinematics_example</name>
   <version>0.0.0</version>
   <description>This package contains an example on how to send Spot an Inverse Kinematic request to move a tool attached to the arm into a given position.</description>
-  <maintainer email="gremigi@theaiinstitute.com">BD AI Institute</maintainer>
+  <maintainer email="gremigi@theaiinstitute.com">AI Institute</maintainer>
```
Comment (amessing-bdai) [inline examples/inverse_kinematics_example/package.xml:7]:
```suggestion
  <maintainer email="engineering@theaiinstitute.com">AI Institute</maintainer>
```

**Review by bhung-bdai — APPROVED** [review]

**Pull request merged**